internal/npm_install/generate_build_file.ts

```diff
@@ -186,24 +186,63 @@ node_module_library(
  * Generates all BUILD & bzl files for a package.
  */
 function generatePackageBuildFiles(pkg: Dep) {
+  // If a BUILD file was shipped with the package, append its contents to the end of
+  // what we generate for the package.
+  let buildFilePath: string|undefined;
+  if (pkg._files.includes('BUILD')) buildFilePath = 'BUILD';
```

Comment:
Could this logic be simplified?
1) copy over all .bzl & BUILD files in the package's tree
2) write out the generated contents to BUILD[.bazel], bin/BUILD[.bazel], index.bzl --- if the file exists append otherwise create?

Reply:
let's take a stab at that in a follow-up, but it's not that simple - I think it's naive to just merge them in some cases.